.github/workflows/stale.yml

```diff
@@ -0,0 +1,33 @@
+name: 'close stale issues/PRs'
+on:
+  schedule:
+    - cron: '* */6 * * *'
+  workflow_dispatch:
```

Comment:
Do we need this trigger?

Reply:
It's helpful for testing or manual kick-off. Resolving unless you think it's harmful to have (footgun?).

Reply:
Ehh, not a fan but I don't have a great argument to remove it so YOLO